samples/pmic/npmx/buck/README.rst

```diff
@@ -0,0 +1,141 @@
+.. _npmx_buck_sample:
+
+npmx: BUCK
+##########
+
+.. contents::
+   :local:
+   :depth: 2
+
+The npmx BUCK sample demonstrates how to control PMIC BUCKs using npmx drivers.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf52840dk_nrf52840, nrf5340dk_nrf5340_cpuapp
+
+The sample also requires an nPM1300 EK.
+
+Overview
+********
+
+This sample uses npmx BUCK driver to:
+  * set output voltage for BUCK converter
+  * set retention voltage for BUCK converter with GPIO
+  * control enable mode for BUCK converter with GPIO
+
+Voltage can be changed only for BUCK1. BUCK2 voltage is set by default to 3.0 V.
```

Comment:
```suggestion
Voltage can be changed only for BUCK 1. BUCK 2 voltage is set by default to 3.0 V.
```

Reply:
And remember, each sentence on a separate line.